src/components/app/details/appConfig/AppConfig.tsx

```diff
@@ -280,35 +314,50 @@ const NextButton: React.FC<{ isCiPipeline: boolean; navItems, currentStageName,
     return null;
 };
 
-function Navigation({ navItems, deleteApp }) {
+function Navigation({ navItems, deleteApp, isCDPipeline}) {
+const pathname = window.location.pathname;
+  const selectedNav = navItems.filter((navItem) => pathname.indexOf(navItem.href)>=0)[0];
     return (
         <>
+            {!isCDPipeline && <div className="help-container">
+                <div>{selectedNav.currentStep}/4 Completed</div>
```

Comment:
use optional (?) or add a check for `selectedNav` wherever it's used to make it fail-safe.